crates/ruff_formatter/src/printer/mod.rs

```diff
@@ -1689,6 +1696,7 @@ two lines`,
         );
     }
 
+    // TODO(cnpryer): Would it make sense to add a test to the printer for measured line suffix?
```

Comment:
These tests tend to be very painful to write. I like to add examples with doctests to the builders (@charliermarsh's dying because of slow doctests but they are very useful for this) to demonstrate the functioning of IR changes.

Reply:
It was a bit tough when I tried adding one. Good to know.

Reply:
I'll mark this as resolved and take note of this. If I land any other printer PRs I'd like to try this again.